redash/query_runner/influx_db.py

```diff
@@ -19,16 +19,32 @@ def _transform_result(results):
     result_rows = []
 
     for result in results:
-        if not result_columns:
-            for c in result.raw['series'][0]['columns']:
-                result_columns.append({ "name": c })
+        for series in result.raw.get('series', []):
+            for column in series['columns']:
+                if column not in result_columns:
+                    result_columns.append(column)
+            tags = series.get('tags', {})
+            for key in tags.keys():
+                if key not in result_columns:
+                    result_columns.append(key)
```

Comment:
Yes, that's possible and it causes problems when attempting to query based on the tag.

See also:
https://docs.influxdata.com/influxdb/v0.11/concepts/schema_and_data_layout/
bottom of the page.